README.md

```diff
@@ -128,24 +125,30 @@ The only thing you need to install is Docker. All of the other dev dependencies
 
     </details>
 
-1. **Build IoT Edge module images**
+4. Build IoT Edge module images.
+
+    ```sh
+    iotedgedev build
+    ```
 
-    `sudo iotedgedev build`
-    
     > This step will build user modules in deployment.template.json targeting amd64 platform.
 
     <details>
     <summary>More information</summary>
 
     1. You will see a "BUILD COMPLETE" for each module and no error messages in the terminal output.
-    1. Open `config/deployment.amd64.json` file, you will see the module image placeholders expanded correctly.
-    1. Run `sudo docker image ls`, you will see the module images you just built.
+    2. Open `config/deployment.amd64.json` file, you will see the module image placeholders expanded correctly.
+    3. Run `sudo docker image ls`, you will see the module images you just built.
 
     </details>
 
-1. **Setup and start the IoT Edge Simulator to run the solution**
+5. Setup and start the IoT Edge Simulator to run the solution

```

Comment:
Please make sure to merge with Eliise's changes that split the VM and simulation experience